test/verify/check-login

```diff
@@ -364,6 +365,52 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
                                     "couldn't parse login input: Malformed input",
                                     "couldn't parse login input: Authentication failed")
 
+    @skipImage("No SELinux", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1804", "ubuntu-2004")
+    @skipImage("No semanage", "fedora-coreos")
+    @unittest.expectedFailure
+    def testSELinuxRestrictedUserSudo(self, with_selinux=True):
+        m = self.machine
+        b = self.browser
+
+        # Configure only sudo, and without using cockpit-askpass
+        m.write("/usr/share/cockpit/shell/override.json", """
+{ "bridges": [
+        {
+            "match": {
+                "superuser": null
+            },
+            "spawn": [
+                "/usr/bin/sudo",
+                "cockpit-bridge",
+                "--privileged"
+            ],
+            "privileged": true
+        }
+
+] }
+""")
+        self.addCleanup(m.execute, "rm /usr/share/cockpit/shell/override.json")
+
+        # No password so that it all works without cockpit-askpass
+        m.write("/etc/sudoers.d/admin", "admin ALL=(ALL) NOPASSWD:ALL")
+        self.addCleanup(m.execute, "rm /etc/sudoers.d/admin")
```

Comment:
Note that this test isn't marked as `@nondestructive`, but it probably should eventually, so that we can use it for downstream gating. So it's fine to keep the cleanups here.

Reply:
The whole `TestLogin` class is `@nondestructive`, isn't that enough?

Reply:
Of course it is, I missed that, sorry :)